.github/workflows/size-impact.yml

```diff
@@ -1,12 +1,9 @@
 name: size-impact
 
-on: pull_request
+on: pull_request_target
```

Comment:
This change is enough to support forks. Other changes are to update @jsenv/file-size-impact to get latest features.